bedrock/firefox/templates/firefox/set-as-default/landing.html

```diff
@@ -16,24 +16,43 @@
   {{ css_bundle('firefox-default-landing') }}
 {% endblock %}
 
+{% block js %}
+  {{ js_bundle("firefox-default-landing") }}
+{% endblock %}
+
+{% if switch('set-as-default-experiment') and  variation == '1' %}
+  {% set landing_header = ftl('set-as-default-landing-you-should-get') %}
+  {% set landing_body = ftl('set-as-default-landing-if-you-recently') %}
+{% elif switch('set-as-default-experiment') and variation == '2' %}
+  {% set landing_header = "Feel safe every time you explore the web" %}
+  {% set landing_body = "Choose the browser that puts your privacy first everyday. Make sure Firefox is your default browser" %}
+{% elif switch('set-as-default-experiment') and variation == '3' %}
+  {% set landing_header = "Choose a browser that supports a better web for you and everyone else" %}
+  {% set landing_body = "Set Firefox back to your default browser and enjoy peace of mind with every link you open." %}
+{% else %}
+    {% set landing_header = ftl('set-as-default-landing-you-should-get') %}
+    {% set landing_body = ftl('set-as-default-landing-if-you-recently') %}
+{% endif %}
+
 {% block content %}
   <main role="main">
-    {% call hero(
-      title=ftl('set-as-default-landing-make-sure-youre-protected'),
```

Comment:
I think you could still use the macro here and set this to `title=landing_header` and `desc=landing_body`

Reply:
i was getting issues yesterday, but works fine this morning, good call out 👍🏻